libs/components/modals/src/lib/modules/modal/modal.service.spec.ts

```diff
@@ -176,9 +176,137 @@ describe('Modal service', () => {
 
   it('should show a modal with a wrapper class', fakeAsync(() => {
     const wrapperClass = 'custom-wrapper-class';
-    openModal(ModalTestComponent, { wrapperClass });
+    const modal = openModal(ModalTestComponent, { wrapperClass });
     applicationRef.tick();
 
     expect(document.body.querySelector(`sky-modal.${wrapperClass}`)).toExist();
+    closeModal(modal);
+  }));
+
+  it('should not hide modal host from screen readers when opening a modal', fakeAsync(() => {
```

Comment:
It might be beneficial to wrap these tests in their own `describe` block so they're easier to digest.